config/settings.py

```diff
@@ -104,7 +104,7 @@
 # Internationalization
 # https://docs.djangoproject.com/en/4.2/topics/i18n/
 
-LANGUAGE_CODE = "pt-br"
+LANGUAGE_CODE = "pt-pt"
```

Comment:
```suggestion
LANGUAGE_CODE = "pt-br"
```

O correto seria o idioma do país que a aplicação vai rodar.